libs/server/StoreWrapper.cs

```diff
@@ -165,6 +166,26 @@ public sealed class StoreWrapper
             run_id = Generator.CreateHexId();
         }
 
+        public byte[] SerializeGarnetObject(IGarnetObject garnetObject)
+        {
+            using var ms = new MemoryStream();
+            var serializer = new GarnetObjectSerializer(customCommandManager);
+            serializer.BeginSerialize(ms);
+            serializer.Serialize(ref garnetObject);
+            serializer.EndSerialize();
+            return ms.ToArray();
+        }
+
+        public IGarnetObject CreateGarnetObject(byte[] data)
+        {
+            using var ms = new MemoryStream(data);
+            var serializer = new GarnetObjectSerializer(customCommandManager);
```

Comment:
I think object serializer needs to be created per session in respserversession as it is not thread safe.

Reply:
The GarnetObjectSerializer is instantiated per call to tackle thread safety.